components/style/properties/properties.mako.rs

```diff
@@ -1181,13 +1181,21 @@ impl ComputedValues {
     #[inline]
     pub fn min_inline_size(&self) -> computed::LengthOrPercentage {
         let position_style = self.get_position();
-        if self.writing_mode.is_vertical() { position_style.min_height } else { position_style.min_width }
+        let size = if self.writing_mode.is_vertical() { position_style.min_height } else { position_style.min_width };
+        match size {
+            computed::MinLength::LengthOrPercentage(lop) => lop,
+            _ => panic!("layout doesn't work!")
```

Comment:
You can undo these changes since you don't actually change servo side, like Manishearth said here https://github.com/servo/servo/pull/14432#r90549771